src/lib/paths.js

```diff
@@ -1,24 +1,20 @@
 const path = require("path");
 
-function getRoutePath(page) {
-  return path.join("/", "page", page);
+function getIpvPagePath(pageId) {
+  return `/ipv/page/${encodeURIComponent(pageId)}`;
 }
 
-function getIpvPagePath(page) {
-  return path.join("/", "ipv", "page", page);
+function getIpvPageTemplatePath(pageId) {
+  return getTemplatePath("ipv", "page", pageId);
 }
 
-function getIpvPageTemplatePath(page) {
-  return path.join("ipv", "page", addNunjucksExt(page));
-}
-
-function addNunjucksExt(path) {
-  return `${path}.njk`;
+function getTemplatePath(...pathComponents) {
+  const pageId = pathComponents.splice(-1, 1);
```

Comment:
I think pageId will be an array, does that work OK with the string interpolation below?

Reply:
It does work ok since an array `.toString()` returns the array values separated by commas

Reply:
Good spot! Yeah it'll work although it's a bit more by luck than judgement 😬